src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" data-qa="big" method="GET">
+      <label for="largeInput" class="large__bar" tabindex="0">
+        <div class="search-icon__large"></div>
+        <input
+          class="input__large"
+          data-qa="keypress"
+          type="text" id="largeInput"
+          placeholder="Try &quot;Los Angeles&quot;"
+          size="100%">
```

Comment:
```suggestion
        <input
          class="input__large"
          data-qa="keypress"
          type="text" 
          id="largeInput"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
![image](https://user-images.githubusercontent.com/96409563/228759754-8e5ccd2a-7cf5-4cdd-879c-97b1102918ef.png)
Better set the size using CSS styles and not size attribute